clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -148,35 +169,32 @@ public void testStartupAndTearDown() throws InterruptedException {
     }
 
     @Test
-    public void testApplicationEvent() {
-        ApplicationEvent e = new PollEvent(100);
-        applicationEventsQueue.add(e);
+    public void testRequestsTransferFromManagersToClientOnThreadRun() {
+        List<Optional<? extends RequestManager>> list = new ArrayList<>();
+        list.add(Optional.of(coordinatorRequestManager));
+        list.add(Optional.of(heartbeatRequestManager));
+        list.add(Optional.of(offsetsRequestManager));
+
+        when(requestManagers.entries()).thenReturn(list);
+        when(coordinatorRequestManager.poll(anyLong())).thenReturn(mock(NetworkClientDelegate.PollResult.class));
         consumerNetworkThread.runOnce();
-        verify(applicationEventProcessor, times(1)).process(e);
+        requestManagers.entries().forEach(rmo -> rmo.ifPresent(rm -> verify(rm).poll(anyLong())));
+        requestManagers.entries().forEach(rmo -> rmo.ifPresent(rm -> verify(rm).maximumTimeToWait(anyLong())));
+        verify(networkClientDelegate).addAll(any(NetworkClientDelegate.PollResult.class));
+        verify(networkClientDelegate).poll(anyLong(), anyLong());
     }
 
-    @Test
-    public void testMetadataUpdateEvent() {
```

Comment:
Seems this test was effectively removed? Why?

Reply:
Refer to previous comment for all event tests that were removed